### PR TITLE
Widget Home Page

### PR DESCRIPTION
**Pull request by rishsingh07**
Styled widget home page.

### CODE REVIEW

**Comment by harshvitra** [comment]
@sagararyal we shifted the page to landing.html for future.

File: public/landing.html [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+				// let circleBGColor = document.getElementById("treeMap").getAttribute("circlebgcolor");
+				// let community = document.getElementById("treeMap").getAttribute("community");
+				// let locale = document.getElementById("treeMap").getAttribute("locale");
+
+				document.getElementById("treeProf").innerHTML = `<tree-profile user=${data.slug}  id="treeProfile"> </tree-profile>`;
+			})
+			.catch((error) => {
+				document.getElementById("error-prof").innerHTML = "Invalid Profile URL";
+			});
+	}
```
Comment (norbertschuler) [inline public/landing.html:361]: This looks like a duplicate of the code before?

Comment (rishsingh07) [reply public/landing.html:361]: Looks similar with different ID's, this one in particular is for tree-profile.

**Comment by norbertschuler** [comment]
- [ ] The code for changing the user of the widgets is not yet compatible with showing the source code to copy:

Bildschirmfoto 2021-03-02 um 12 19 49

**Comment by harshvitra** [comment]
Fix user picture design

Screenshot 2021-03-03 at 1 04 14 PM

**Comment by norbertschuler** [comment]
@rishsingh07 Is this "Ready for review"?

**Comment by sagararyal** [comment]
- [ ] Default profile for now: https://www1.plant-for-the-planet.org/t/matthias-betz

**Pull request merged**